jdi-light/src/main/java/com/epam/jdi/light/driver/WebDriverFactory.java

```diff
@@ -32,91 +30,138 @@
 public class WebDriverFactory {
     private WebDriverFactory() { }
 
-    public static MapArray<String, JFunc<WebDriver>> DRIVERS
-        = new MapArray<>(DEFAULT_DRIVER, () -> initDriver(CHROME));
-    public static boolean SINGLE_THREAD = false;
-    private static MapArray<String, WebDriver> RUN_DRIVERS = new MapArray<>();
-    private static Safe<MapArray<String, WebDriver>> THREAD_RUN_DRIVERS
-        = new Safe<>(MapArray::new);
+    public static boolean MULTI_THREAD = false;
+    public static MapArray<String, JFunc<WebDriver>> DRIVERS = new MapArray<>();
+    public static final MapArray<String, WebDriver> RUN_DRIVERS = new MapArray<>();
+    private static final Safe<MapArray<String, WebDriver>> THREAD_RUN_DRIVERS = new Safe<>(MapArray::new);
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6397461)